.github/workflows/add-issues-to-projects.yml

```diff
@@ -0,0 +1,17 @@
+name: Auto Assign to Project
+
+on:
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue to project
```

Comment:
Job name says "Add issue to project" but trigger is for pull_requests

Reply:
Yep. The question if the same file should be used with multiple steps when I move on to issues, then issues was the most common name (since a PR is a issue). Or, should we have one file per type?

Reply:
Ah, sorry - the job. Yes, agree.